router/src/main/kotlin/io/moia/router/RequestHandler.kt

```diff
@@ -119,8 +120,9 @@ abstract class RequestHandler : RequestHandler<APIGatewayProxyRequestEvent, APIG
     /**
      * Customize the format of an unprocessable entity error
      */
-    open fun createUnprocessableEntityErrorBody(error: UnprocessableEntityError): Any =
-        error
+    open fun createUnprocessableEntityErrorBody(error: UnprocessableEntityError): Any = listOf(error)
```

Comment:
Add deprecated please and call the list method from this on by default

Reply:
we decided to remove the (hook) method completely since it is an open method